tuxemon/menu/quantity.py

```diff
@@ -107,6 +117,9 @@ def on_open(self) -> None:
         self.menu_items.arrange_menu_items()
 
     def initialize_items(self) -> Generator[MenuItem[None], None, None]:
+        # Show the money by using the method from the parent class:
+        yield from super().show_money()
```

Comment:
I think this will work if you do self.show_money rather than super().show_money.

Can you check?